src/object_store/src/object/s3.rs

```diff
@@ -392,6 +393,44 @@ impl ObjectStore for S3ObjectStore {
         Ok(())
     }
 
+    /// Deletes the objects with the given paths permanently from the storage. If an object
+    /// specified in the request is not found, it will be considered as successfully deleted.
+    ///
+    /// Uses AWS' DeleteObjects API. See [AWS Docs](https://docs.aws.amazon.com/AmazonS3/latest/API/API_DeleteObjects.html) for more details.
+    async fn delete_objects(&self, paths: &[String]) -> ObjectResult<()> {
+        // AWS restricts the number of objects per request to 1000.
+        const MAX_LEN: usize = 1000;
```

Comment:
Shall we reduce the limit so that it is smaller than "AWS  strict limit" ?

Reply:
Is there an advantage to do that? What number would you suggest?